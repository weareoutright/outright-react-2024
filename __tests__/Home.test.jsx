// HomePage.test.js
import { render } from "@testing-library/react";
import { expect, it, describe } from "vitest";
import Home from "../app/page.jsx";
import ErrorBoundary from "./ErrorBoundary";

describe("HomePage component", () => {
  it("matches the snapshot", () => {
    const { asFragment } = render(
      <ErrorBoundary>
        <Home />
      </ErrorBoundary>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
