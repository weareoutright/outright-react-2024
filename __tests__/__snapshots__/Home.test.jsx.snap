// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HomePage component > matches the snapshot 1`] = `
<DocumentFragment>
  <div>
    Error loading component
  </div>
</DocumentFragment>
`;
